commit 5506984bcd69a9fba9f86f9952d6ca6518f2c746
Author: Wanderley Caloni <wanderley.caloni@bitforge.com.br>
Date:   Tue Sep 17 13:34:39 2019 -0300

    Including config file openssl.cnf in installation.

diff --git a/ports/openssl-windows/portfile.cmake b/ports/openssl-windows/portfile.cmake
index 3506be9ab..22dfe8274 100644
--- a/ports/openssl-windows/portfile.cmake
+++ b/ports/openssl-windows/portfile.cmake
@@ -171,11 +171,11 @@ file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 file(REMOVE
     ${CURRENT_PACKAGES_DIR}/debug/bin/openssl.exe
     ${CURRENT_PACKAGES_DIR}/debug/openssl.cnf
-    ${CURRENT_PACKAGES_DIR}/openssl.cnf
 )

 file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/tools/openssl/)
 file(RENAME ${CURRENT_PACKAGES_DIR}/bin/openssl.exe ${CURRENT_PACKAGES_DIR}/tools/openssl/openssl.exe)
+file(RENAME ${CURRENT_PACKAGES_DIR}/openssl.cnf ${CURRENT_PACKAGES_DIR}/tools/openssl/openssl.cnf)

 vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/openssl)
>openssl genrsa -out subdomain.domain.com.key 1024

WARNING: can't open config file: some_path/openssl.cnf
Loading 'screen' into random state - done
Generating RSA private key, 1024 bit long modulus
.........++++++
.........................................++++++
unable to write 'random state'
e is 65537 (0x10001)
